packages/runtime/src/main/resources/META-INF/native-image/jni-config.json

```diff
@@ -421,5 +421,9 @@
 {
   "name":"sun.nio.ch.FileChannelImpl",
   "fields":[{"name":"fd"}]
+},
+{
+  "name":"com.sun.jna.Native",
+  "methods":[{"name":"dispose","parameterTypes":[] }, {"name":"fromNative","parameterTypes":["com.sun.jna.FromNativeConverter","java.lang.Object","java.lang.reflect.Method"] }, {"name":"fromNative","parameterTypes":["java.lang.Class","java.lang.Object"] }, {"name":"fromNative","parameterTypes":["java.lang.reflect.Method","java.lang.Object"] }, {"name":"nativeType","parameterTypes":["java.lang.Class"] }, {"name":"toNative","parameterTypes":["com.sun.jna.ToNativeConverter","java.lang.Object"] }]
```

Comment:
one or more methods in this were not registered in the JNA feature upstream (which we will need to fix separately), namely:
```json
{"name":"fromNative","parameterTypes":["java.lang.reflect.Method","java.lang.Object"] }
```